src/network/quic.rs

```diff
@@ -64,10 +66,12 @@ pub struct QuicTwoPartyNet {
     send_stream: Option<SendStream>,
     /// The receive side of the bidirectional stream
     recv_stream: Option<RecvStream>,
+    /// The phantom on the curve group
+    _phantom: PhantomData<C>,
```

Comment:
i've never fully understood when a situation calls for the use of `PhantomData` - here, is it because we want to be able to `impl Stream` where `type Item = Result<NetworkOutbound<C>, MpcNetworkError>` - i.e., we use the `PhantomData` to use the same generic type parameter in `NetworkOutbound`?

Reply:
Yeah exactly, we want to be able to use the generic in implementations; we cannot attach the generic elsewhere in the `impl Stream` because that would change the trait signature, so we add it to the struct. Of course, generics in a struct must be bound to some member of the struct, so we add a phantom for this purpose.